Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/README.md

```diff
@@ -0,0 +1,24 @@
+## Overview
```

Comment:
we might want to add the sub feeds details here, like you have in the mimecast-node